libs/entitlement/qa/entitled-plugin/src/main/java/org/elasticsearch/entitlement/qa/entitled/EntitledActions.java

```diff
@@ -19,4 +24,7 @@ static void System_clearProperty(String key) {
         System.clearProperty(key);
     }
 
+    public static UserPrincipal getFileOwner(Path path) throws IOException {
+        return Files.getOwner(path);
```

Comment:
How does this work? I don't see you granting this in the policy file.

Reply:
It doesn't actually do anything, yet.